refs/types.go

```diff
@@ -183,6 +183,16 @@ func (s *SubnetID) GetValue() uint32 {
 	return 0
 }
 
+// IsZeroSubnet returns true iff the SubnetID refers to zero subnet.
+func IsZeroSubnet(id *SubnetID) bool {
+	return id.GetValue() == 0
+}
+
+// MakeZeroSubnet makes the SubnetID to refer to zero subnet.
+func MakeZeroSubnet(id *SubnetID) {
```

Comment:
I dont really see a value in this wrapper, `NewSubnetID` or sth like this should return the default subnet.

Reply:
We already use it in `IterateSubnets`

Reply:
I mean yes, but we can use `NewSubnetID` there instead. IMO missing value should be equal to the default subnet for all API version, so I can't see what are the benefits of `MakeSubnetZero` over `NewSubnetID`.

Reply:
`MakeSubnetZero` doesn't allocates new ID, so it is more basic. If you need to allocate new ID and make it zero, then you can just use `var` declaration.